### ISSUE TITLE
Card border not displaying in Android

### ISSUE THREAD

**Issue by anam-hossain** [GeekyAnts/NativeBase#1818]
Card border not displaying in Android but in IOS its working fine.

My setup:
```json
 "native-base": "^2.4.1",
 "react": "^16.3.0-alpha.1",
  "react-native": "0.55",
  "react-native-easy-grid": "^0.1.17",
```

```jsx
          <Card>
            <CardItem header bordered>
              <Text style={{ textAlign: 'center' }}>Title</Text>
            </CardItem>
            <CardItem bordered>
              <Body>
                <Text>
                  Body text
                </Text>
              </Body>
            </CardItem>
          </Card>
```

**Comment by akhil-geekyants** [comment]
@anam-hossain this will be fixed in the upcoming release.

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.4.3](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.4.3)

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.4.4](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.4.4)

**Comment by artus** [comment]
I'm having this issue on native-base 2.13.15:

![cardlist](https://user-images.githubusercontent.com/22665347/104850106-67638180-58ed-11eb-8b4f-642779979e09.jpg)

These are 3 `<Card>` components with a `<CardItem header bordered` header, but the second one does not show a border.
